diff -u old/condor.pm new/condor.pm
--- old/condor.pm	2009-09-23 13:28:56.000000000 -0500
+++ new/condor.pm	2009-09-23 13:32:35.000000000 -0500
@@ -7,6 +7,7 @@
 use Globus::Core::Paths;
 
 use Config;
+use Globus::GRAM::JobManager::condor_groupacct;
 
 # NOTE: This package name must match the name of the .pm file!!
 package Globus::GRAM::JobManager::condor;
@@ -297,6 +298,8 @@
 	$submit_attrs_string = '';
     }
 
+    $group = Globus::GRAM::JobManager::condor_groupacct::obtain_condor_group(\@environment, \$self);
+
     # Create script for condor submission
     $script_filename = $self->job_dir() . '/scheduler_condor_submit_script';
 
@@ -352,6 +355,10 @@
 # End VDT patch
     print SCRIPT_FILE "#Extra attributes specified by client\n";
     print SCRIPT_FILE "$submit_attrs_string\n";
+    if ($group) {
+	$name = getpwuid($>);
+	print SCRIPT_FILE "+AccountingGroup = \"$group.$name\"\n" if $group;
+    }
 
     for (my $i = 0; $i < $description->count(); $i++) {
         if ($multi_output) {
